Branch-SDK/Branch-SDK/BranchUniversalObject.h

```diff
@@ -8,6 +8,7 @@
 
 #import <Foundation/Foundation.h>
 #import "Branch.h"
+#import <CoreSpotlight/CoreSpotlight.h>
```

Comment:
We can't require CoreSpotlight as a dependency for BUO. Developers don't add this framework unless they are interested in leveraging CoreSpotlight.